README.md

```diff
@@ -20,12 +20,27 @@
 [Citing](#Citing)
 
 # How to install
-Biosiglive can be installed from anaconda or from the sources.
+Biosiglive can be installed from anaconda, pip or from the sources.
 ## Installing from anaconda
 You can install biosiglive from anaconda by running the following command :
 ```bash
 conda install -c conda-forge biosiglive
 ```
+## Installing from pip
+You can install biosiglive from pip by running the following command :
+```bash
+pip install biosiglive matplotlib
+```
```

Comment:
Not sure why you need to add `matplotlib` here? Isn't it listed as a dependency?